students/JasneetChandok/Lesson03/slicinglab.py

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env python3.6
+
+'''Module 3 - Slicing Lab Excercise'''
+
+
+def first_last_exchange(seq):
+	'''First and last item exchanged'''
+	new_sequence = seq[-1:] + seq[1:-1] + seq[:1]
+	return new_sequence
+
+
+def every_other(seq):
+	'''Every other item removed'''
+	new_sequence = seq[::2]
+	return new_sequence
+
+
+def first4_last4_eveyother(seq):
+	'''First 4 and the last 4 items removed, and then every other item in the remaining sequence'''
+	new_sequence = seq[4:-4:2]
+	return new_sequence
+
+
+def elements_reversed(seq):
+	'''elements reversed (just with slicing)'''
+	new_sequence = seq[::-1]
+	return new_sequence
+
+
+def first_last_middle_thrid(seq):

```

Comment:
This is very close except you are taking the last third and reversing it. We want last third - first third - middle third. You will want to update this to fit as expected. Your test case will also need to be updated